.github/workflows/tests.yml

```diff
@@ -15,6 +15,8 @@ jobs:
         go-version: [1.10]
     steps:
     - uses: actions/checkout@v1
+    - name: Kustomize and diff
```

Comment:
cool.  it seems this runs as part of this PR too.

Reply:
Yeah that was a successful execution.